content/applications/websites/livechat.rst

```diff
@@ -98,10 +98,10 @@ settings for the live chat window.
 
 .. _livechat/livechat-button:
 
-Livechat button
-~~~~~~~~~~~~~~~
+Livechat button section
```

Comment:
Adding "section" in the headings over and over across all docs is unnecessary. It's implied since it's a mid-tier heading that it's a section label; I think the way you were doing it before Jess was better

Reply:
cc: @ksc-odoo

Reply:
@StraubCreative gotcha. I'll be sure to not leave such a suggestion moving forward. Thanks for pointing this out. Really appreciate it. 🙏